src/app/globals.css

```
@@ -1,26 +1,191 @@
+  outline: none;
+}
+.google-btn:hover {
+  box-shadow: 0 4px 14px rgba(0,0,0,.10);
+}
+
+.google-logo {
+  background: #fff;
+  border-radius: 50%;
 }
 
+@import "tailwindcss";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)